schemas/aerleon-definition-yaml.schema.json

```diff
@@ -0,0 +1,114 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$comment": "https://aerleon.readthedocs.io/en/latest/reference/naming/",
+  "title": "Aerleon Definition File",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "networks": {
+      "description": "An Aerleon definition file defines named networks and services which can be referenced by Aerleon Policy files. Networks are lists of IP addresses or CIDR IP address ranges. Services are lists of port/protocol pairs (e.g. tcp/80).\nDefinition files must include a \"networks\" section, a \"services\" section, or both.\nNetworks and services may use composition in their definitions. Include another definition by creating an item containing the \"name\" property.",
```

Comment:
Isn't this the help section for networks? Should it talk about services here?

Reply:
Correct, and the standard style would call for this text only discussing networks. In this version of the file I tried to use a different style where all top-level keys have copies of the same overview text. But that is not really in line with the standard style so I will change it in a future PR.